sdk/python/feast/ui_server.py

```diff
@@ -101,6 +101,14 @@ def start_server(
     get_registry_dump: Callable,
     project_id: str,
     registry_ttl_sec: int,
+    root_path: Optional[str] = "",
```

Comment:
```suggestion
    root_path: Optional[str] = None,
```

Reply:
the default value of `root_path` in both uvicorn and FastAPI is empty string "", I'm afraid `None` would break the internal libraries

Reply:
just checked, everything goes well with None